scope.go

```diff
@@ -247,10 +315,13 @@ func (s *scope) SubScope(prefix string) Scope {
 }
 
 func (s *scope) Capabilities() Capabilities {
-	if s.reporter == nil {
-		return capabilitiesNone
+	if s.reporter != nil {
+		return s.reporter.Capabilities()
 	}
-	return s.reporter.Capabilities()
+	if s.cachedReporter != nil {
+		return s.cachedReporter.CachedCapabilities()
+	}
+	return capabilitiesNone
```

Comment:
If we were using the baseStatsReporter approach this could also just be:
```go
if s.baseReporter != nil [
  return s.baseReporter.Capabilities()
}
return capabilitiesNone
```